standard_plus_issue/models/standard_plus_issue.py

```diff
@@ -153,6 +154,9 @@ def action_set_submitted(
         self.ensure_one()
         self.send_by_email(self.prepare_mail_attachments(), template_ref)
         self.state = 'submitted'
+        self.message_post(
+            body=_('Email sent to ') + self.property_support_email,
+            subject=None, message_type='notification')
```

Comment:
We will need to talk about how emails are used in this module.

You should not have post a notification. The email sent should appear inside the mail thread.
Also, the address used by the support should be a partner, not a bare email address.

When someone logs a note inside the issue, as a support team member, I want to be advised.

Reply:
That's not in the scope of the change request.
Your change requests are valuable and we should put them in the next release.